dev/io.openliberty.springboot.fat30.mbean.app/src/main/java/io/openliberty/springboot/support/version30/test/mbean/app/TestMBeanInterface.java

```diff
@@ -0,0 +1,16 @@
+package io.openliberty.springboot.support.version30.test.mbean.app;
+
+import javax.management.AttributeList;
+import javax.management.DynamicMBean;
+import javax.management.MBeanException;
+import javax.management.NotCompliantMBeanException;
+import javax.management.ReflectionException;
+
+public interface TestMBeanInterface{
```

Comment:
I was expecting this to be called `InterfaceTestMXBean` like the 20 application interface.